pageserver/src/page_service.rs

```diff
@@ -258,6 +258,8 @@ async fn page_service_conn_main(
     socket.set_timeout(Some(std::time::Duration::from_millis(socket_timeout_ms)));
     let socket = std::pin::pin!(socket);
 
+    fail::fail_point!("ps::connection-start::pre-login");
```

Comment:
nit: the rest of the pageserver failpoints don't have a `ps::` prefix (they're implicitly scoped because when we enable them we're talking to the pageserver API)